.pre-commit-config.yaml

```diff
@@ -1,17 +1,18 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+exclude: |
+  (?x)^(
+    docs/source/conf.py|
+    pcdsdevices/_version.py|
+  )$
+
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
     hooks:
     -   id: no-commit-to-branch
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
-        exclude: |
-                (?x)^(
-                    pcdsdevices/tests/test_lens_sets/.*|
-                    .*\.ui$
-                )$
```

Comment:
This needs to be reverted or else you get a changed file every time you run the test suite that gets changed back again when you run the pre-commit

Reply:
I'm glad you're cautiously checking these - I got lost in rabbit holes and neglected the real intent of the PR at some point.

Reply:
Reverted below